src/order/rel_iso.lean

```diff
@@ -65,9 +153,11 @@ here but mimic its signature by using `⦃e₁ e₂⦄`. -/
 theorem coe_fn_inj : ∀ ⦃e₁ e₂ : r ↪r s⦄, (e₁ : α → β) = e₂ → e₁ = e₂
 | ⟨⟨f₁, h₁⟩, o₁⟩ ⟨⟨f₂, h₂⟩, o₂⟩ h := by { congr, exact h }
 
+/-- Identity map is a relation embedding. -/
 @[refl] protected def refl (r : α → α → Prop) : r ↪r r :=
 ⟨embedding.refl _, λ a b, iff.rfl⟩
 
+/-- Composition of two relation embeddings is a relation embedding. -/
 @[trans] protected def trans (f : r ↪r s) (g : s ↪r t) : r ↪r t :=
 ⟨f.1.trans g.1, λ a b, by rw [f.2, g.2]; simp⟩
```

Comment:
Ideally these two definitions are also renamed `id`/`comp`

Reply:
I'm not sure about this, because in the library for general embeddings, they're called `embedding.refl` and `embedding.trans`. They are maps in a category, but that category is pretty close to being a partial ordering.

Reply:
Interesting, I thought for embeddings we would also use `id`/`comp`. If it is already established to use `refl`/`trans` then we should of course also do that here.